### DISCUSSION TITLE
FileNotFoundError: Plug-in Flake8-pyproject could not find custom configuration file

### DISCUSSION THREAD

**Discussion by Smit-Parmar** [idom-team/idom#1021] [Problem]
Getting FileNotFoundError while running `hatch run lint-py` command. 

Error only where there is not linting needed. 
Throwing error on command below
`flake8 --toml-config /home/dir1 name with space/reactpy/pyproject.toml .`
Works fine when running
`flake8 --toml-config pyproject.toml .`

**Comment by rmorshea** [comment]
Fixed by: https://github.com/reactive-python/reactpy/pull/1023
